src/js/components/FilterCriteria.jsx

```diff
@@ -91,11 +94,32 @@ class FilterCriteria extends Component {
                 onChange={(ev) => onChangeValue(i, ev.target.value)}
               />
               {(() => {
+                if (filters.length !== 1) {
+                  return (
+                    <FloatingActionButton
+                      mini={true}
+                      primary={true}
+                      style={{
+                        ...style,
+                        margin: '1em 0 0 1em'
+                      }}
+                      onTouchTap={(ev) => onRemove(i, ev)}
```

Comment:
yeah. im not perfect lol xD. they should all try to work as similarly to regular handlers as possible. if you want to change the others to work similarly to regular event handlers, that would be awesome and welcome